### PR TITLE
feat: CMake port media/replicator

### PR DESCRIPTION
**Pull request by zdanek**
Issue #1047 (CMake porting)

### CODE REVIEW

File: packager/media/replicator/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Copyright 2022 Google Inc. All rights reserved.
+#
+# Use of this source code is governed by a BSD-style
+# license that can be found in the LICENSE file or at
+# https://developers.google.com/open-source/licenses/bsd
+
+add_library(media_replicator STATIC
+        replicator.cc)
```
Comment (joeyparrish) [inline packager/media/replicator/CMakeLists.txt:8]: style nit: align with previous line arguments

File: packager/media/replicator/CMakeLists.txt [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+target_link_libraries(media_replicator
+        glog
```
Comment (joeyparrish) [inline packager/media/replicator/CMakeLists.txt:10]: style nit: align with previous line arguments

File: packager/media/replicator/CMakeLists.txt [hunk 3]
```diff
@@ -0,0 +1,11 @@
+# Copyright 2022 Google Inc. All rights reserved.
```
Comment (joeyparrish) [inline packager/media/replicator/CMakeLists.txt:1]: nit: Google LLC

We recently had to do a mass update to correct this in old files, so let's not add the old name in any new files.  Thanks!

**Review by joeyparrish — CHANGES_REQUESTED** [review]

File: packager/media/replicator/CMakeLists.txt (hunk 1 above)
Comment (joeyparrish) [reply packager/media/replicator/CMakeLists.txt:8]: Actually, no, I was wrong.  This should be 4-space indentation.  I was misremembering the convention we were using in CMakeLists.

File: packager/media/replicator/CMakeLists.txt (hunk 2 above)
Comment (joeyparrish) [reply packager/media/replicator/CMakeLists.txt:10]: My mistake, 4 spaces, please

**Review by joeyparrish — APPROVED** [review]

**Comment by joeyparrish** [comment]
I took the liberty of making the few minor stylistic changes I wanted in CMakeLists.txt, and I fixed merge conflicts with media/origin.  Thanks!

File: packager/media/replicator/CMakeLists.txt (hunk 3 above)
Comment (zdanek) [reply packager/media/replicator/CMakeLists.txt]: This is great - I made the same mistake you did a few weeks ago 😅

**Pull request merged**